### ISSUE TITLE
Running into `error: no known class method for selector` during xcodebuild

### ISSUE THREAD

**Issue by thepoppingone** [emartech/react-native-emarsys-sdk#39]
```
/Users/macmini/repos/clone/macmini-mobile-app/ios/Pods/RNEmarsysWrapper/ios/RNEmarsysEventHandler.m:21:18: error: property 'notificationEventHandler' not
      found on object of type 'id<EMSPushNotificationProtocol> _Nonnull'
    Emarsys.push.notificationEventHandler = ^(NSString * _Nonnull eventName, NSDictionary<NSString *,id> * _Nullable payload) {
                 ^
/Users/macmini/repos/clone/macmini-mobile-app/ios/Pods/RNEmarsysWrapper/ios/RNEmarsysEventHandler.m:27:32: error: assigning to 'id<EMSEventHandler>' from
      incompatible type 'void (^)(NSString * _Nonnull __strong, NSDictionary<NSString *,id> * _Nullable __strong)'
    Emarsys.inApp.eventHandler = ^(NSString * _Nonnull eventName, NSDictionary<NSString *,id> * _Nullable payload) {
                               ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/macmini/repos/clone/macmini-mobile-app/ios/Pods/RNEmarsysWrapper/ios/RNEmarsysEventHandler.m:33:44: error: assigning to
      'id<EMSEventHandler> _Nullable' from incompatible type 'void (^)(NSString * _Nonnull __strong, NSDictionary<NSString *,id> * _Nullable __strong)'
    Emarsys.push.silentMessageEventHandler = ^(NSString * _Nonnull eventName, NSDictionary<NSString *,id> * _Nullable payload) {
                                           ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/macmini/repos/clone/macmini-mobile-app/ios/Pods/RNEmarsysWrapper/ios/RNEmarsysEventHandler.m:39:35: error: assigning to
      'id<EMSEventHandler> _Nullable' from incompatible type 'void (^)(NSString * _Nonnull __strong, NSDictionary<NSString *,id> * _Nullable __strong)'
    Emarsys.geofence.eventHandler = ^(NSString * _Nonnull eventName, NSDictionary<NSString *,id> * _Nullable payload) {
                                  ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/macmini/repos/clone/macmini-mobile-app/ios/Pods/RNEmarsysWrapper/ios/RNEmarsysEventHandler.m:45:40: error: assigning to
      'id<EMSEventHandler> _Nullable' from incompatible type 'void (^)(NSString * _Nonnull __strong, NSDictionary<NSString *,id> * _Nullable __strong)'
    Emarsys.onEventAction.eventHandler = ^(NSString * _Nonnull eventName, NSDictionary<NSString *,id> * _Nullable payload) {
                                       ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
5 errors generated.
```

Did anyone face this issue? It was working okay before the SDK and Wrapper updates.
Tested on M1 Mac and Intel Mac both throws the same error on the `RNEmarsysEventHandler.m` file

**Comment by thepoppingone** [comment]
Seems like this issue goes away after upgrade to the latest SDK on Xcode 11.4

**Comment by thepoppingone** [comment]
Hi, I need to reopen this issue, as we are unable to update to the latest SDK due to application code changes.

**Comment by thepoppingone** [comment]
@eduzatoni Hi we found out that `react-native-emarsys-wrapper/ios/RNEmarsysWrapper.podspec` file is causing this issue.
https://github.com/emartech/react-native-emarsys-sdk/commit/55f9541011fc7fd612b473cb9cf70e19740e2ce0#diff-21e1acd870fbc60ca7e8d7e98fab9c390885710028f95a2e919eb68f6c7b7dd0

As our node package version is locked at #1.2.0, the podspec is still referring to the master version
`- s.source = { :git => "https://git@github.com/emartech/react-native-emarsys-sdk.git", :tag => "master" }`

I see that you have updated to 1.4.0 20 days ago in the latest repo, is it possible for you to update this in the released package of #1.2.0 to 1.2.0 in the podspec as well?

This is breaking our builds on newly onboarded machines as even though the versions are locked for node modules to 1.2.0 the SDK in fact is pulling the latest one which has breaking changes. 

Thanks and I look forward to your reply!
